recipe/build.sh

```diff
@@ -1,6 +1,36 @@
-set -ex
+#! /bin/bash
+
+set -eu
+
 ./bootstrap.sh
 
+# We have pre-downloaded the data files.  If bin/libpostal_data is run
+# during the build it will update these files if necessary which you
+# can verify by removing the --disable-data-download from configure,
+# below.
+
+mkdir -p ${PREFIX}/share/libpostal_data
+mv __upstream/share/libpostal_data/libpostal ${PREFIX}/share/libpostal_data
```

Comment:
Should we use `install -v` instead of `mv`?

Reply:
This is a hierarchy of files which `install` doesn't like.  I've switched to `cp -rp` as it might be marginally better than `mv` in corner cases.